lua/wire/server/wirelib.lua

```diff
@@ -1216,9 +1216,15 @@ function WireLib.AddOutputAlias( class, old, new )
 	ENT_table.OutputAliases[old] = new
 end
 
+local function effectiveMass(ent)
+	if not isentity(ent) then return 1 end
+	if ent:IsWorld() then return 99999 end
+	if not IsValid(ent) or not IsValid(ent:GetPhysicsObject()) then return 1 end
```

Comment:
It will _never_ become invalid unless someone overwrites `GetPhysicsObject` to remove the physics object, which is both stupid, and something we should not have to check for.